### DISCUSSION TITLE
使用3.x版本，用实验室服务器训练，但是默认使用显卡0，如何在这个版本指定显卡？

### DISCUSSION THREAD

**Discussion by ghost** [open-mmlab/mmdetection#9965] [Q&A]
实验室一台服务器有很多卡，但是3.x版本里我没有找到可以指定显卡号进行训练的代码位置，该怎么解决这一问题？

**Comment by hhaAndroid** [comment]
https://mmyolo.readthedocs.io/zh_CN/dev/common_usage/specify_device.html
